src/linear_algebra/affine_space/independent.lean

```diff
@@ -548,5 +548,16 @@ begin
   exact s.centroid_eq_iff h₁ h₂
 end
 
+/-- Two simplices with the same points have the same centroid. -/
+lemma centroid_eq_of_range_eq {n : ℕ} {s₁ s₂ : simplex k P n}
+  (h : set.range s₁.points = set.range s₂.points) :
+  finset.univ.centroid k s₁.points = finset.univ.centroid k s₂.points :=
```

Comment:
Can you make this the stronger statement
```suggestion
lemma centroid_eq_of_range_eq {n : ℕ} {p₁ p₂ : ι → P}
  (h : set.range p₁ = set.range p₂) :
  finset.univ.centroid k p₁.points = finset.univ.centroid k p₂ :=
```

Reply:
See `centroid_eq_of_inj_on_of_image_eq` for a general statement. The point of this particular one is to specialise that to the case of a simplex.